Extensions/tweaks.js

```diff
@@ -152,6 +152,11 @@ XKit.extensions.tweaks = new Object({
 			default: false,
 			value: false
 		},
+		"wide_sources": {
+			text: "Increase max width of post sources to avoid shortening",
+			default: true,
+			value: false
```

Comment:
I honestly have no idea why this is useful, but `value` should always match `default`.